packages/documentation-site/patternfly-docs/pages/home.js

```diff
@@ -112,6 +117,40 @@ const AggregateCards = () => {
   );
 };
 
+const FeaturedBlogCard = ({postData, idx}) => {
+  const { URL, author, imageURL, length, title } = postData;
+  const cardId = `featured-blog-post-${idx}`;
+  const actionId = `card-article-input-${idx}`;
+
+  return (
+    <Card id={cardId} component="div" isClickable key={idx} aria-labelledby={`featured-blog-post-${idx}-title`}>
+      <CardHeader
+        className="v5-featured-posts-card-header-img"
+        selectableActions={{
+          to: URL,
+          selectableActionId: actionId,
+          selectableActionAriaLabelledby: cardId,
```

Comment:
This should probably be set to the `featured-blog-post-${idx}-title` text instead (assuming there will always be a title). Right now, at least in VO, these cards just come up as "radio button X of Y" in the VO rotor menu; updating one of them to the `featured-blog-post-0-title` causes it to render with a more meaningful description.

The same would probably apply to the AggregateCards as well

Reply:
Thanks, this makes sense - made this update here and in AggregateCards as well